### DISCUSSION TITLE
Implement hlsSourceRedirect

### DISCUSSION THREAD

**Discussion by dorinclisu** [bluenviron/mediamtx#710] [General]
Currently we have sourceRedirect with path: redirect, but this only works for rtsp, and is ignored by HLS requests.
Could a similar mechanism be implemented for HLS, and provide the additional path field `hlsSourceRedirect`?

This would be necessary to maintain request parity, otherwise HLS requests are going to end up on servers that are not sourcing the respective path.

**Comment by aler9** [comment]
Hello, open a feature request by following instructions.

[a comment by an automated account is not shown]
